src/theme/raw.rs

```diff
@@ -915,6 +920,9 @@ pub(crate) enum ParseColorError {
     #[error("invalid hex color: {0}")]
     Hex(#[from] FromHexError),
 
+    #[error("Hex color should only be 3 or 6 long, got hex string of length {0}.")]
```

Comment:
Can you use no capital on Hex and no period at the end like the rest of the errors?